rc/filetype/kotlin.kak

```diff
@@ -56,6 +56,7 @@ provide-module kotlin %§
 add-highlighter shared/kotlin regions
 add-highlighter shared/kotlin/code default-region group
 add-highlighter shared/kotlin/string region %{(?<!')"} %{(?<!\\)(\\\\)*"} fill string
+add-highlighter shared/kotlin/character region %{(?<!")'} %{(?<!\\)(\\\\)*'} fill string
```

Comment:
I guess this can be simplified a bit: we don't need the check for `"` because the string region is checked earlier, and there can be at most one backslash.

```suggestion
add-highlighter shared/java/character region %{'} %{(?<!\\)'} fill string
```

I don't know if that's really better, it's simpler anyway

Reply:
could you add the same change to `java.kak`? I think they are the same here